Cargo.toml

```diff
@@ -4,9 +4,16 @@ version = "0.1.0"
 authors = ["Robin Lovelace <rob00x@gmail.com>"]
 edition = "2018"
 
+[lib]
```

Comment:
The changes here are based on https://rustwasm.github.io/book/game-of-life/hello-world.html. They make it so that this is both a regular Rust library crate and something that can be compiled as a WASM library